ccdImports/configFiles/CCD_Probate_Caveat/AuthorisationCaseEvent.json

```diff
@@ -120,6 +121,7 @@
   {"CaseTypeID": "Caveat", "CaseEventID": "amendCaveatForCaveatNotMatched", "UserRole": "caseworker-probate-caseadmin", "CRUD": "CRUD"},
   {"CaseTypeID": "Caveat", "CaseEventID": "addCommentForCaveatRaised", "UserRole": "caseworker-probate-caseadmin", "CRUD": "CRUD"},
   {"CaseTypeID": "Caveat", "CaseEventID": "findMatchedCaseForImport", "UserRole": "caseworker-probate-caseadmin", "CRUD": "CRUD"},
+  {"CaseTypeID": "Caveat", "CaseEventID": "addCommentForCaveatMatching", "UserRole": "caseworker-probate-caseadmin", "CRUD": "CRUD"},
```

Comment:
This event should be accessible by all roles. In Caveats caseadmin is the most restrictive other roles have access to all states/events at the moment.